shared/teams/add-members-wizard/confirm.tsx

```diff
@@ -5,33 +5,72 @@ import * as Container from '../../util/container'
 import * as Constants from '../../constants/teams'
 import * as Types from '../../constants/types/teams'
 import * as TeamsGen from '../../actions/teams-gen'
-import * as RouteTreeGen from '../../actions/route-tree-gen'
+import * as RPCGen from '../../constants/types/rpc-gen'
+import {appendNewTeamBuilder, appendTeamsContactsTeamBuilder} from '../../actions/typed-routes'
 import capitalize from 'lodash/capitalize'
 import {FloatingRolePicker} from '../role-picker'
 import {ModalTitle} from '../common'
+import {pluralize} from '../../util/string'
 
 const AddMembersConfirm = () => {
   const dispatch = Container.useDispatch()
   const nav = Container.useSafeNavigation()
 
-  const {teamID, addingMembers} = Container.useSelector(s => s.teams.addMembersWizard)
+  const {teamID, role, addingMembers} = Container.useSelector(s => s.teams.addMembersWizard)
   const teamname = Container.useSelector(s => Constants.getTeamMeta(s, teamID).teamname)
   const noun = addingMembers.length === 1 ? 'person' : 'people'
 
-  const onClose = () => dispatch(RouteTreeGen.createClearModals())
-  const onBack = () => dispatch(nav.safeNavigateUpPayload())
+  const onLeave = () => dispatch(TeamsGen.createCancelAddMembersWizard())
+
+  const [waiting, setWaiting] = React.useState(false)
+  const [error, setError] = React.useState('')
+  const addMembers = Container.useRPC(RPCGen.teamsTeamAddMembersMultiRoleRpcPromise)
+  const onComplete = () => {
+    setWaiting(true)
+    addMembers(
+      [
+        {
+          sendChatNotification: true,
+          teamID,
+          users: addingMembers.map(member => ({
+            assertionOrEmail: member.assertion,
+            role: RPCGen.TeamRole[role || 'writer'], // TODO Y2K-1560 handle individual roles
+          })),
+        },
+      ],
+      _ => {
+        // TODO handle users not added?
```

Comment:
doing this in a different PR? or do you think we should ask design for a design?

Reply:
we should probably ask for designs

Reply:
I left a zeplin comment